notebooks/en/semantic_cache_chroma_vector_database.ipynb

```diff
@@ -0,0 +1,1389 @@
+{
```

Comment:
<p>Correct the question asked: "How work a vaccine" is grammatically wrong.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
changed to: 

How do vaccines work?